src/comp/ProjectTag.tsx

```diff
@@ -3,13 +3,20 @@ import * as styles from "css/comp/ProjectTag.module.css";
 
 export type ProjectTagProp = {
   tag_id: string;
+  onClick: (e: React.MouseEvent<HTMLParagraphElement>) => void;
```

Comment:
```suggestion
  onClick?: (e: React.MouseEvent<HTMLParagraphElement>) => void;
```
このままでは指定しないパターンを受け付けないのでは(憶測)